config/kubernetes/org.yaml

```diff
@@ -865,6 +865,7 @@ members:
 - saiyam1814
 - saiyan86
 - sakshamsharma
+- salaxander
```

Comment:
Looks like @salaxander need another sponsor for joining K8s org. 

/hold

Reply:
added @karenhchu as a sponsor

Reply:
Also added @jeremyrickard

Reply:
I approve :)